source/common/request_info/dynamic_metadata_impl.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <map>
+
+#include "envoy/request_info/dynamic_metadata.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace RequestInfo {
+
+class DynamicMetadataImpl : public DynamicMetadata {
+public:
+  DynamicMetadataImpl() {}
+  ~DynamicMetadataImpl() override{};
```

Comment:
Don't think we need either of these empty ones.

Reply:
Done.